airflow/providers/ssh/operators/ssh.py

```diff
@@ -156,6 +167,8 @@ def raise_for_status(self, exit_status: int, stderr: bytes, context=None) -> Non
         if context and self.do_xcom_push:
             ti = context.get("task_instance")
             ti.xcom_push(key="ssh_exit", value=exit_status)
+        if self.skip_on_exit_code is not None and exit_status in self.skip_on_exit_code:
+            raise AirflowSkipException(f"SSH command returned exit code {self.skip_on_exit_code}. Skipping.")
```

Comment:
```suggestion
            raise AirflowSkipException(f"SSH command returned exit code {exit_status}. Skipping.")
```

I think we might want to log the raised exit code instead of the allowed exit code

Reply:
Fixed